crates/ruff_linter/src/rules/pylint/snapshots/ruff_linter__rules__pylint__tests__PLC0206_dict_index_missing_items.py.snap

```diff
@@ -0,0 +1,67 @@
+---
+source: crates/ruff_linter/src/rules/pylint/mod.rs
+---
+dict_index_missing_items.py:9:1: PLC0206 Extracting value from dictionary without calling `.items()`
+   |
+ 8 |   # Errors
+ 9 | / for instrument in ORCHESTRA:
+10 | |     print(f"{instrument}: {ORCHESTRA[instrument]}")
+   | |___________________________________________________^ PLC0206
+11 |   
+12 |   for instrument in ORCHESTRA:
```

Comment:
It would be quite useful to have a infrastructure to provide additional details here similar to Biome. That would avoid highlighting the entire `for` statement and instead we could highlight individual pieces of code with details.